docs/tutorials/random-sample-generator.md

```diff
@@ -0,0 +1,271 @@
+
+```yaml
+source:
+  ref: generate_sample_demo
+  fields:
+    - name_parts: [ email ]
+      type: varchar
+      required: true
+    - name_parts: [ gebruikersnaam ]
+      type: varchar
+      required: true
+    - name_parts: [ organisatie ]
+      type: varchar
+      required: true
+    - name_parts: [ klantnummber ]
+      type: varchar
+      required: true
+    - name_parts: [ bankrekening ]
+      type: varchar
+      required: false
+
+additional_system_instructions:
+- don't forget to add the csv column headers
+- please return 5 result rows
+```
+
+</details>
+
+<details>
+<summary><code>complex-instructions.yaml</code></summary>
+In this file, we've added some more system instructions, to enrich the generated sample data.
+
+```yaml
+source:
+  ref: generate_sample_demo
+  fields:
+    - name_parts: [ email ]
+      type: varchar
+      required: true
+    - name_parts: [ gebruikersnaam ]
+      type: varchar
+      required: true
+    - name_parts: [ organisatie ]
+      type: varchar
+      required: true
+    - name_parts: [ klantnummber ]
+      type: varchar
+      required: true
+    - name_parts: [ bankrekening ]
+      type: varchar
+      required: false
+
+additional_system_instructions:
+  - don't forget to add the csv column headers
+  - please return 20 result rows
+  - if you recognize something as being an email, please use email domains in europe
+  - if you recognize something as a customer number please generate 7 digits between 1000000 and 2999999
+  - for recognized bank accounts use european style IBAN
+```
+</details>
+
+## Generating the Data Policy
+
+### Tutorial video
+
+{% embed url="https://youtu.be/_1y6l4Sz2Y8" %}
+
+### Running PACE
+
+Make sure your current working directory is the same as the directory you've set up in the previous section. Start the containers by running:
+
+```bash
+docker compose up
+```
+
+There should be quite a bit of logging, ending in the banner of the PACE app booting.
+
+### Invoking the plugin
+
+First we prepare some instructions:
+
+```yaml
+source:
+  ref: generate_sample_demo
+  fields:
+    - name_parts: [ email ]
+      type: varchar
+      required: true
+    - name_parts: [ gebruikersnaam ]
+      type: varchar
+      required: true
+    - name_parts: [ organisatie ]
+      type: varchar
+      required: true
+    - name_parts: [ klantnummber ]
+      type: varchar
+      required: true
+    - name_parts: [ bankrekening ]
+      type: varchar
+      required: false
+additional_system_instructions:
+  - don't forget to add the csv column headers
+  - please return 5 result rows
+```
+In the same directory, execute the following PACE CLI command:
+
+```bash
+pace  invoke plugin openai GENERATE_SAMPLE_DATA --payload instructions.yaml
+```
+
+This will take a little while (around 20 seconds during our testing). If OpenAI replied with a valid Data Policy, it will be printed to your terminal. The output should look similar to this:
```

Comment:
` If OpenAI replied with a valid Data Policy, it will be printed to your terminal.` -> not a `Data Policy`, but if GPT is able to generate a result.